components/authentication-framework/org.wso2.carbon.identity.application.authentication.endpoint/src/main/webapp/login.jsp

```diff
@@ -131,7 +131,7 @@
             if (reCaptchaEnabled) {
         %>
         <script src='<%=
-        (request.getParameter("reCaptchaAPI"))%>'></script>
+        Encode.forJavaScriptAttribute(request.getParameter("reCaptchaAPI"))%>'></script>
```

Comment:
shouldn't this be 'forHtmlAttribute' ? as the src here is a html attribute

Reply:
@isuruirj We very much appreciate the contribution. 

The correct way is
```
Encode.forJavaScriptSource
```
Done at the commit 681c16f0b3bb90f626c0489f17599d4a7a3ebf55
https://github.com/wso2/carbon-identity-framework/commit/681c16f0b3bb90f626c0489f17599d4a7a3ebf55